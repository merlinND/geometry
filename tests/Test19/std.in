C myCircle 0 0 1
OA myCircle myRectangle myLine myOA2 myLine
EXIT
