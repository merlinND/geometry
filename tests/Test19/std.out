C myCircle 0 0 1
OK
ERR
